data/hai missions.txt

```diff
@@ -1053,3 +1053,245 @@ mission "Hai Honeymoon"
+				accept
+			label lie
+			`	You go on about how Nanachi was the one who taught you how to fly a ship, and how famous she is amongst the Hai pilots, but as he looks at you with great disdain, he interrupts your stories. "Oh of course, an experienced captain that has been running around the spaceport for hours and can't even figure out the job board! I felt like I could maybe trust you to take care of her and the cargo, buy if you won't at least be honest with a fellow captain I can't put you in charge of a mission like this. Now get out of here, I have better things to do than waste my time listening to nonsense!" You and Nanachi continue to look for someone willing to give her a job, but after hours it's clear your best bet was with that man. Unable to find any other way to help her, you apologize and wish her the best of luck.`
+				decline
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog `	You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
+	on complete
+		conversation
+			`	Shortly after her ship lands, you meet up with Nanachi, who is doing her best to hide her excitement of being on another planet for the first time, gawking at some Hai military ships in particular. "So, how did you like your first time flying a ship?" you ask, to try and snap her out of it. "Oh um, I had a bit of trouble with the controls at first since they're a bit different from the ones in Hai ships, but I figured it out. I've always dreamt of being on other worlds, exploring and learning everything about them, I'm glad I'm finally able to do it." You help her get the crates out of the cargo hold and contact the client. Her share of the payment is small, as the client transfers most of the money to that merchant, but that doesn't make Nanachi any less excited about getting her first paycheck.`
+			`	"So, want to go to the spaceport and try to get a job on your own? I'll help you again in case you can't do it yet." you say. "Just give me an hour or two, back home our shipyard is kind of small, and I've never really gotten a chance to see a Shield Beetle up close, and there are many undergoing repairs here, I'll just go get a better look at them, I swear I'll be quick." You two agree to meet in the spaceport in a few hours, maybe this time she will do fine on her own.`
```

Comment:
Again with needing a new line because of the person speaking switching. I'd also add a new line after Nanachi's dialog ends. i.e.
`"So..." you say.`
`"Just... quick"`
`You... own.`

```suggestion
			`	"So, want to go to the spaceport and try to get a job on your own? I'll help you again in case you can't do it yet." you say. "Just give me an hour or two. Back home our shipyard is kind of small. I've never really gotten a chance to see a Shield Beetle up close, and there are many undergoing repairs here. I'll just go get a better look at them, I swear I'll be quick." You two agree to meet in the spaceport in a few hours, maybe this time she will do fine on her own.`
```